packages/federatorai/overlay/questions.yaml

```diff
@@ -0,0 +1,79 @@
+questions:
+#image configurations
+- variable: defaultImage
+  default: true
+  description: "Use default Federator.ai image or specify a custom one"
+  label: Use Default Federator.ai Image
+  type: boolean
+  show_subquestion_if: false
+  group: "Container Images"
+  subquestions:
+  - variable: image.repository
+    default: "quay.io/prophetstor/federatorai-operator-ubi"
+    description: "Federator.ai Operator image name"
+    type: string
+    label: Federator.ai Operator Image Name
+  - variable: image.tag
+    default: "v4.5.1-ga"
+    description: "Federator.ai Operator image tag"
+    type: string
+    label: Federator.ai Operator Image Tag
+#service configurations
+- variable: federatorai.imageLocation
+  default: "quay.io/prophetstor"
+  description: "Service containers image location"
+  type: string
+  required: true
+  label: Federator.ai imageLocation
+- variable: federatorai.version
+  default: "v4.5.1-ga"
+  description: "Service containers version"
+  type: string
+  required: true
+  label: Service Containers Image Tag
+- variable: services.dashboardFrontend.nodePort
+  required: true
+  default: "31012"
+  description: "The port where the Federator.ai Dashboard listens to"
+  type: string
+  label: Federator.ai Dashboard Port
+- variable: services.rest.nodePort
+  required: true
+  default: "31011"
+  description: "The port where the Federator.ai REST listens to"
+  type: string
+  label: Federator.ai REST Port
+- variable: federatorai.persistence.enabled
+  default: true
+  description: "Enable persistent volume for Federator.ai"
+  type: boolean
+  required: true
+  label: Federator.ai Persistent Volume Enabled
+  show_subquestion_if: true
+  group: "PV Settings"
+  subquestions:
+  - variable: federatorai.persistence.storageClass
+    default: ""
+    description: "If undefined or set to null, using the default storageClass. Defaults to null."
+    type: storageclass
+    label: Storage Class for Federator.ai
+  - variable: federatorai.persistence.storages.logStorage.size
+    default: "2Gi"
+    description: "Log volume size"
+    type: string
+    label: Log Volume Size
+  - variable: federatorai.persistence.aiCore.dataStorage.size
+    default: "10Gi"
+    description: "AICore data volume Size"
+    type: string
+    label: AICore Data Volume Size
+  - variable: federatorai.persistence.influxdb.dataStorage.size
+    default: "100Gi"
+    description: "Influxdb data volume Size"
+    type: string
+    label: Influxdb Data Volume Size
+  - variable: federatorai.persistence.fedemeterInfluxdb.dataStorage.size
+    default: "10Gi"
+    description: "Fedemeter influxdb data volume Size"
+    type: string
+    label: Fedemeter Influxdb Data Volume Size
```

Comment:
Just FYI, these are good, no need to add groups to these because the parent question has a group which applies to all subquestions.

Reply:
Added `group: <group-name>`.